WebViewControl.Avalonia/BaseControl.cs

```diff
@@ -1,31 +1,18 @@
-﻿using System;
-using Avalonia.Controls;
-using Avalonia.Interactivity;
+﻿using Avalonia.Controls;
 using Avalonia.LogicalTree;
 
-namespace WebViewControl {
+namespace WebViewControl;
```

Comment:
revert... we should do this for all files in a diff PR

Reply:
Fine by me. In the IDE projects we migrate them on the fly when we touch them